packages/host/app/components/ai-assistant/skill-menu/index.gts

```diff
@@ -0,0 +1,224 @@
+import { fn } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+import onClickOutside from 'ember-click-outside/modifiers/on-click-outside';
+import { restartableTask } from 'ember-concurrency';
+
+import { AddButton } from '@cardstack/boxel-ui/components';
+import { cn, not } from '@cardstack/boxel-ui/helpers';
+
+import { chooseCard, skillCardRef } from '@cardstack/runtime-common';
+
+import CardPill from '@cardstack/host/components/card-pill';
+
+import type {
+  SkillField,
+  SkillCard,
+} from 'https://cardstack.com/base/skill-card';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    skills: SkillField[];
+    attachSkill: (skill: SkillCard) => void;
+  };
+}
+
+export default class AiAssistantSkillMenu extends Component<Signature> {
```

Comment:
There will be another feature that uses almost identical design–permissions in the AI panel:

<img width="668" alt="image" src="https://github.com/cardstack/boxel/assets/273660/988c7dae-0e36-4ea3-b74d-3b09b0f2f7d4">

Perhaps it would be good to implement this menu in a way where it can be reused in both places.